plugins/catalog-unprocessed-entities/src/index.ts

```diff
@@ -17,3 +17,4 @@ export {
   catalogUnprocessedEntitiesPlugin,
   CatalogUnprocessedEntitiesPage,
 } from './plugin';
+export { UnprocessedEntitiesContent } from './components/UnprocessedEntities';
```

Comment:
I think this needs to be exposed in the plugin.ts as a componentEntity too to make use of it when imported as external dependency.